### PR TITLE
Backport sponsor & news changes from Portland to Prague

### PR DESCRIPTION
**Pull request by ericholscher**
Refs #462

### CODE REVIEW

**Comment by ericholscher** [comment]
Getting weird CSS issues on the peopletext image. I _love_ CSS.

File: docs/conf/prague/2018/index.rst [hunk 1]
```diff
@@ -8,27 +8,27 @@
...
+      <section>
+      <div class="content">
+
+
+
+.. postlist:: 10
+   :date: %B %d, %Y
+   :format: {title} - {date}
+   :list-style: none
+   :tags: prague-2018
```
Comment (plaindocs) [inline docs/conf/prague/2018/index.rst:21]: You could use `{{shortcode}}-{{year}}` here if you wanted. probably. :-p

**Comment by plaindocs** [comment]
Thanks a ton for getting to this.

I still get weird green lines on the Sponsorship page. https://deploy-preview-466--dentist-canary-64055.netlify.com/conf/prague/2018/sponsors/

**Comment by plaindocs** [comment]
Same in Portland actually :/

**Comment by ericholscher** [comment]
Bleh CSS

**Comment by ericholscher** [comment]
Fixed the worst css. Merging this to get it out.

**Pull request merged**